lib/ruby_smb/field/smb_gea_list.rb

```diff
@@ -4,7 +4,7 @@ module Field
     # [2.2.1.2.1.1 SMB_GEA_LIST](https://msdn.microsoft.com/en-us/library/ff359447.aspx)
     class SmbGeaList < BinData::Record
       endian :little
-      uint32  :size_of_list, label: 'Size of List in Bytes', initial_value: -> { gea_list.do_num_bytes }
+      uint32  :size_of_list, label: 'Size of List in Bytes', initial_value: -> { self.do_num_bytes }
```

Comment:
This change was needed to follow the protocol: https://msdn.microsoft.com/en-us/library/ff359447.aspx